tests/Security/BruteForceLoginFormAuthenticatorTest.php

```diff
@@ -28,15 +30,15 @@
 class BruteForceLoginFormAuthenticatorTest extends TestCase
 {
     private BruteForceLoginFormAuthenticator $authenticator;
-    /** @var HttpUtils&\PHPUnit\Framework\MockObject\MockObject */
+    /** @var HttpUtils&MockObject */
     private $httpUtils;
```

Comment:
Could use this I believe?

```suggestion
    private HttpUtils&MockObject $httpUtils;
```